### PR TITLE
Infra/ change _.invoke to optional chaining

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Use optional chaining instead of _.invoke 

## Changelog
Use optional chaining instead of _.invoke

### CODE REVIEW

File: src/components/tabController/PageCarousel.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ class PageCarousel extends PureComponent {
 
   scrollToPage = (pageIndex: number) => {
     const {pageWidth} = this.context;
-    const node = _.invoke(this.carousel, 'current.getNode');
+    const node = this.carousel.current?.getNode();
```
Comment (ethanshar) [inline src/components/tabController/PageCarousel.tsx:37]: Should it be 
```suggestion
    const node = this.carousel.current?.getNode?.();
```

File: src/components/colorPicker/ColorPickerDialog.tsx [hunk 2]
```diff
@@ -212,14 +212,14 @@ class ColorPickerDialog extends PureComponent<Props, State> {
...
-      _.invoke(this.props, 'onSubmit', hex, this.getTextColor(hex));
+      this.props.onSubmit?.(hex, this.getTextColor(hex));
       this.onDismiss();
     }
   };
 
   onDismiss = () => {
     this.resetValues();
-    _.invoke(this.props, 'onDismiss');
+    this.props.onDismiss?.({});
```
Comment (ethanshar) [inline src/components/colorPicker/ColorPickerDialog.tsx:222]: Why are u passing here an empty object?

File: src/components/actionSheet/index.tsx [hunk 3]
```diff
@@ -139,8 +139,8 @@ class ActionSheet extends Component<ActionSheetProps> {
   }
 
   onOptionPress(optionIndex: number) {
-    _.invoke(this.props, `options[${optionIndex}].onPress`);
-    _.invoke(this.props, 'onDismiss');
+    this.props.options?.[optionIndex].onPress;
```
Comment (ethanshar) [inline src/components/actionSheet/index.tsx:142]: You missed calling the function here
```suggestion
    this.props.options?.[optionIndex]?.onPress?.();
```

File: src/components/actionSheet/index.tsx [hunk 4]
```diff
@@ -139,8 +139,8 @@ class ActionSheet extends Component<ActionSheetProps> {
...
+    this.props.onDismiss?.({});
```
Comment (ethanshar) [inline src/components/actionSheet/index.tsx:143]: Why are u passing an empty object here?

File: src/components/colorPicker/ColorPickerDialog.tsx (hunk 2 above)
Comment (lidord-wix) [reply src/components/colorPicker/ColorPickerDialog.tsx:222]: The onDismiss props expect to get props
Changing it to be optional..

**Review by ethanshar — APPROVED** [review]

**Pull request merged**